### PR TITLE
Honor, Fire, Burn Thy Fear - the Hegemony Corvette

### PR DESCRIPTION
**Pull request by RustingWithYou**
Adds a new overmap ship, the Izweski Hegemony Corvette. The ship has six ghostroles, making up a small crew of the Izweski Navy. There are three non-whitelisted crewman roles, as well as a Vaurca whitelist-locked K'lax auxiliary and an Unathi whitelist-locked captain and warpriest role. 

To fill this out, and help distinguish it from the Kataphracts, new voidsuits have been added for the Izweski military, as well as a new variant of energy shield and energy glaive, and an energy rifle. All sprites done by @CAPTGecko#9618. Though I have modified it, the original map design was made by @Kaizr#1111.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

File: code/modules/projectiles/projectile/beams.dm [hunk 1]
```diff
@@ -54,6 +54,14 @@
 	damage = 40
 	armor_penetration = 20
 
+/obj/item/projectile/beam/midlaser/hegemony
+	damage = 40
+	armor_penetration = 20
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/projectile/beams.dm:59]: the description on the rifle says "A special crystal used in its design allows it to penetrate armor with pinpoint accuracy." but it actually has less AP and more damage than the normal laser rifle? 

moreover this has heavy laser cannon damage while having the same firerate/shots as the normal laser rifle, I'm not sure I agree with that

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/projectile/beams.dm (hunk 1 above)
Comment (RustingWithYou) [reply code/modules/projectiles/projectile/beams.dm:59]: I have never coded a gun before, and have no idea what is balanced or not. I'll tweak it appropriately.

**Review by WickedCybs — CHANGES_REQUESTED** [review]

File: maps/away/ships/hegemony/hegemony_ghostroles.dm [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+	name = "Hegemony Navy Crewman"
+	desc = "You are a sworn warrior of the Izweski Hegemony Navy, your life and honor pledged to Hegemon Not'zar. Abide by the Warrior's Code, and follow the orders of your superior officers. Remember, you serve the Izweski Hegemony."
+	tags = list("External")
+	spawnpoints = list("hegemony_crew")
+	req_perms = null
+	max_count = 2
+	uses_species_whitelist = FALSE
+	mob_name_prefix = "Cn. "
+	mob_name_pick_message = "Pick an Unathi last name."
+
```
[the remark this answers is not in the document]
Comment (RustingWithYou) [reply maps/away/ships/hegemony/hegemony_ghostroles.dm]: welcome messages have been added

File: code/modules/projectiles/projectile/beams.dm (hunk 1 above)
Comment (RustingWithYou) [reply code/modules/projectiles/projectile/beams.dm]: Gun has been tweaked to have lower damage and higher AP

**Review by WickedCybs — APPROVED** [review]

**Review by WickedCybs — CHANGES_REQUESTED** [review]
Something I missed at first. You need to make sure the shuttle can cycle foreign atmospheres, which means adding a vent or vents inside the airlock that lead to an external one outside the ship that it can get or expel atmosphere from. Look at something like the intrepid to get an idea of how that's done or ask questions over the coder channel if need be.

**Review by WickedCybs — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**